components/admin_console/admin_definition.jsx

```diff
@@ -266,30 +287,39 @@ const AdminDefinition = {
         icon: 'fa-users',
         sectionTitle: t('admin.sidebar.userManagement'),
         sectionTitleDefault: 'User Management',
-        system_user_detail: {
-            url: 'user_management/user/:user_id',
-            schema: {
-                id: 'SystemUserDetail',
-                component: SystemUserDetail,
-            },
-        },
+        isHidden: it.all(
+            it.not(it.userHasReadPermissionOnResource('user_management')),
+            it.not(it.userHasReadPermissionOnResource('user_management.users')),
+            it.not(it.userHasReadPermissionOnResource('user_management.groups')),
+            it.not(it.userHasReadPermissionOnResource('user_management.teams')),
+            it.not(it.userHasReadPermissionOnResource('user_management.channels')),
+            it.not(it.userHasReadPermissionOnResource('user_management.permissions')),
```

Comment:
wont all of the above return the same value as `(it.userHasReadPermissionOnResource('user_management')` ?

Reply:
I guess its for the future if we want to split up access based on subsection right? makes sense

Reply:
We only want to hide the section if the user does not have read to any of the subsections.